modules/search/instant-search/components/search-box.jsx

```diff
@@ -36,6 +42,21 @@ const SearchBox = props => {
 				type="search"
 				value={ props.query }
 			/>
+			{ ! props.widget && (
+				/* Using role='button' rather than button element so we retain control over styling */
+				<div
+					role="button"
+					onClick={ props.toggleFilters }
+					onKeyDown={ props.toggleFilters }
+					tabIndex="0"
+					className="jetpack-instant-search__box-filter-icon"
+				>
+					<Gridicon icon="filter" alt="Search filter icon" />
+					<span class="screen-reader-text">
+						{ props.showFilters ? __( 'Hide filters' ) : __( 'Show filters ' ) }
+					</span>
```

Comment:
Wouldn't the `alt` and `screen-reader-text` be redundant to the SVG title that currently reads `Toggle search filters`?

Reply:
I think the 'show' and 'hide' text might be more useful because it reflects the current state of button. I'll add a `aria-hidden` to the icon to avoid duplication.